kiwi/boot/arch/arm/oemboot/suse-SLES13/config.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#================
+# FILE          : config.sh
+#----------------
+# PROJECT       : openSUSE KIWI Image System
+# COPYRIGHT     : (c) 2006 SUSE LINUX Products GmbH. All rights reserved
+#               :
+# AUTHOR        : Marcus Schaefer <ms@suse.de>
+#               :
+# BELONGS TO    : Operating System images
+#               :
+# DESCRIPTION   : configuration script for SUSE based
+#               : operating systems
+#               :
+#               :
+# STATUS        : BETA
+#----------------
+#======================================
+# Functions...
+#--------------------------------------
+test -f /.kconfig && . /.kconfig
+test -f /.profile && . /.profile
+
+#======================================
+# Greeting...
+#--------------------------------------
+echo "Configure image: [$kiwi_iname]..."
+
+#======================================
+# SuSEconfig
+#--------------------------------------
+suseConfig
+
+#==========================================
+# setup gfxboot
+#------------------------------------------
+suseGFXBoot openSUSE grub
```

Comment:
The theme name provided as argument to the method is just a default theme name. The value only applies if the XML config does not provide a theme name. However if no theme name is provided the next generation kiwi also generates a boot setup based on textmode and thus the value has no effect. So this is basically for historical reasons and to be sure there is always a default theme name.

The code around suseGFXBoot could be refactored but so far I just left it untouched because it just works :)